index.html

```diff
@@ -0,0 +1,423 @@
+            class="link"
+            target="_blank"
+            rel="noreferrer noopener"
+          >
+            Get my resume
+          </a>
+          <img class="about-me-icon" src="./media/svg/icon-2.svg" alt="Icon" />
+        </section>
+        <hr />
+        <div class="about-me-section stacks">
+          <ul>
+            <li class="languages list-title">
+              Languages
+              <ul>
+                <li>JavaScript</li>
+                <li>TypeScript</li>
+                <li>Ruby</li>
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>Sass</li>
+              </ul>
+            </li>
+          </ul>
+          <ul>
+            <li class="frameworks list-title">
+              Frameworks
+              <ul>
+                <li>React & Redux</li>
+                <li>Material-UI</li>
+                <li>Jest</li>
+                <li>Bootstrap</li>
+                <li>Ruby on Rails</li>
+                <li>RSPec</li>
+              </ul>
+            </li>
+          </ul>
+          <ul>
+            <li class="skills list-title">
+              Skills
+              <ul>
+                <li>GitHub</li>
+                <li>CodePen</li>
+                <li>GitLab</li>
+                <li>Terminal</li>
+                <li>WSL</li>
+                <li>Remote Pair-Programming</li>
+                <li>Teamwork</li>
+                <li>Mentoring</li>
+                <li>Code Reviews</li>
+                <li>Mobile/Responsive Development</li>
+              </ul>
+            </li>
+          </ul>
+        </div>
+      </div>
+      <section class="contact-section">
+        <div class="contact-section-container">
+          <div class="contact-section-head">
+            <p class="contact-section-info">
+              I'm always interested in hearing about new projects, so if you'd
+              like to chat please get in touch.
+            </p>
+          </div>
+          <form
+            class="contact-form"
+            action="https://formspree.io/f/mrgrkgvy"
+            method="POST"
+          >
+            <fieldset class="fieldset">
+              <ul>
+                <li>
+                  <input
+                    type="text"
+                    name="name"
+                    id="name"
+                    maxlength="30"
+                    placeholder="Full name"
+                    required
+                  />
+                </li>
+                <li>
+                  <input
+                    type="text"
+                    name="email"
+                    id="email"
+                    placeholder="Email address"
+                  />
+                </li>
+
+                <li>
+                  <textarea
+                    id="message"
+                    class="textarea"
+                    cols="30"
+                    rows="10"
+                    maxlength="550"
```

Comment:
- [ ] Thank you for increasing the  maxlength attribute of the `textarea`. I believe you've increased it too much to 550. If you can please adjust this to 500 that would be great 👍. 

_It's important to note that 550 is a good value in general for a textarea in this case but since you already have set guidelines with this project it's best to follow them 😄._